### DISCUSSION TITLE
Qselect on v2, @input-value doesn't work.

### DISCUSSION THREAD

**Discussion by curlylazy** [quasarframework/quasar#9754] [General - Components / Directives / etc] [answered]
i use qselect on quasar v2 (with latest release), when i use @input-value to triggred change event, it seem doesn't work, can somebody help me ?, is this bug or something.

thanks.

**Comment by metalsadman** [comment]
@curlylazy what's not working? https://codepen.io/metalsadman/pen/yLMWXmv?editors=101. please make a reproduction codepen.

**Reply by curlylazy** [reply]
woah thank you for reply my answer,
i'm not using "use-input" and "fill-input" maybe this is cause @input-value not triggred, i will try again.

thank you @metalsadman.

**Reply by metalsadman** [reply]
yes it won't work if you don't have `use-input`, since q-select won't be using an input tag, no way for the listener to fire if that's the case.

**Comment by hawkeye64** [comment]
@curlylazy Perhaps you are thinking of `@update:model-value`? In which case, refresh yourself with the upgrade guide.

**Reply by hawkeye64** [reply]
nvm, I see there is an `input-value` event
![image](https://user-images.githubusercontent.com/10262924/123549122-d35fd680-d724-11eb-958c-25a157a5881e.png)

**Reply by hawkeye64** [reply]
I made a pen -- seems to be broken: https://codepen.io/Hawkeye64/pen/poPzxvp?editors=1010

**Reply by hawkeye64** [reply]
Figured it out and fixed the pen.

**Comment by hawkeye64** [comment, marked as answer]
@curlylazy You have to use `input-value` event along with prop `use-input`

**Reply by curlylazy** [reply]
thank for reply my question, yes in quasar v1 i just use @input to detect changed value on Qselect, but now it fixed, after i add 
use-input, i forgot to read the new documentation off Quasar V2 about Qselect, my bad..
